packages/next/build/webpack/loaders/next-swc-loader.js

```diff
@@ -191,3 +191,6 @@ export default function swcLoader(inputSource, inputSourceMap) {
       }
     )
 }
+
+// accept Buffers instead of strings
+export const raw = true
```

Comment:
What is this?
It seems like it's not used by anywhere.

Reply:
https://webpack.js.org/api/loaders/#raw-loader